src/manage.py

```diff
@@ -53,18 +54,38 @@ def seed():
         "Testing": test_data,
         "Staging": production_data
     }
-    if environment == "Testing" or \
-        prompt_bool("\n\n\nThis operation will remove all existing data."
-                    " Are you sure you want to continue?"):
+
+    if environment.lower() in ["production", "staging"]:
```

Comment:
Are there plans to remove this once production is up? I have mad jitters about leaving it in.